website/public/js/controllers/settingsCtrl.js

```diff
@@ -206,10 +206,10 @@ habitrpg.controller('SettingsCtrl',
       var baseCap = 25;
       var gemCapExtra = User.user.purchased.plan.consecutive.gemCapExtra;
       // @TODO: What are these magic numbers? 3? 5?
-      var blocks = Content.subscriptionBlocks[subscription.key].months / 3 * 5;
```

Comment:
Is the new commit what you were after?  I had a look at adding them as vars in Content and referencing them here, but soon realized that I don't understand how script/src is converted to src.  

I reckon it would be better added to Content, but I didn't want to mess things up.